bindings/jupyter-modules/jupyter-ma-causal/src/containers/index.js

```diff
@@ -58,7 +58,9 @@ class App extends Component {
 
             <LineIndicatorContainer index={i} />
           </div>
-          <div style={{position: 'relative', width: columnWidth}}>
+          <div
+            style={{position: 'relative', width: columnWidth, marginTop: 13}}
+          >
             {' '}
```

Comment:
Is it really needed? Can't we just add marginLeft to get the space?

Reply:
this is typo ...